src/test/java/org/cactoos/io/TeeOutputTest.java

```diff
@@ -50,83 +50,70 @@
-                        new OutputTo(copy)
+            () -> new TextOf(
+                new Sticky(
+                    new TeeInput(
+                        new InputOf("Hello, товарищ!"),
+                            new TeeOutput(
+                            new OutputTo(baos),
+                            new OutputTo(new ByteArrayOutputStream())
+                        )
                     )
                 )
             ),
-            new TextHasString(
-                new MatcherOf<>(
-                    str -> {
-                        return new String(
-                            copy.toByteArray(), StandardCharsets.UTF_8
-                        ).equals(str);
-                    }
-                )
+            new TextIs(
+                new TextOf(baos::toByteArray, StandardCharsets.UTF_8)
             )
-        );
+        ).affirm();
     }
 
     @Test
     public void copiesWithWriter() {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        final ByteArrayOutputStream copy = new ByteArrayOutputStream();
-        MatcherAssert.assertThat(
+        new Assertion<>(
             "Can't copy Output with writer",
-            new TextOf(
-                new TeeInput(
-                    new InputOf("Hello, товарищ! writer"),
-                    new TeeOutput(
-                        new OutputTo(baos),
-                        new WriterTo(copy)
+            () -> new TextOf(
+                new Sticky(
+                    new TeeInput(
+                        new InputOf("Hello, товарищ! writer"),
+                        new TeeOutput(
+                            new OutputTo(baos),
+                            new WriterTo(new ByteArrayOutputStream())
+                        )
                     )
                 )
             ),
-            new TextHasString(
-                new MatcherOf<>(
-                    str -> {
-                        return new String(
-                            copy.toByteArray(), StandardCharsets.UTF_8
-                        ).equals(str);
-                    }
-                )
+            new TextIs(
+                new TextOf(baos::toByteArray, StandardCharsets.UTF_8)
             )
-        );
+        ).affirm();
     }
 
     @Test
     public void copiesWithWriterAndCharset() {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        final ByteArrayOutputStream copy = new ByteArrayOutputStream();
-        MatcherAssert.assertThat(
+        new Assertion<>(
             "Can't copy Output with writer and charset",
-            new TextOf(
-                new TeeInput(
-                    new InputOf("Hello, товарищ! writer and charset"),
-                    new TeeOutput(
-                        new OutputTo(baos),
-                        new WriterTo(copy),
-                        StandardCharsets.UTF_8
+            () -> new TextOf(
+                new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above.

Reply:
@Iprogrammerr the same above.